src/components/GridTemplates/DetailsPage1/DetailsPage1.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * @file Details page template #1.
```

Comment:
Quick note -- I threw allllll of the subcomponents in the same file for this initial review. But let me know if it is too much to look at, and I can split them up. I do plan to split them out eventually, but I didn't want to do that right now while the naming structure is still up in the air.

Reply:
That's exactly what I did for the `Shell` spike, so I completely understand!